server/src/controllers/christmas-tree.es6

```diff
@@ -62,11 +84,99 @@ christmasTree.getForests = (req, res) => {
+    });
+};
+
+const translateFromApplicationToPayment = application => {
+  const result = {
+    paymentAmount: application.totalCost,
+    formName: '',
+    applicantName: application.firstName + ' ' + application.lastName,
+    applicantEmailAddress: application.emailAddress,
+    selectedOption: '',
+    description: '',
+    amountOwed: application.totalCost,
+    permitId: application.permitId
+  };
+  return result;
 };
 
+christmasTree.create = (req, res) => {
+  treesDb.christmasTreesPermits.create(translatePermitFromClientToDatabase(req.body))
+    .then(response => {
+      
+      const tcsAppID = vcapConstants.payGovAppId;
+                        
+      const xmlTemplate = [ 
+                          { 
+                            'S:Envelope':[
+                              { _attr: { 'xmlns:S': 'http://schemas.xmlsoap.org/soap/envelope/' } },
+                              { 'S:Header': [] },
+                              {
+                                'S:Body': [ 
+                                  { 
+                                    'ns2:startOnlineCollection': [ 
+                                      { _attr: { 'xmlns:ns2': 'http://fms.treas.gov/services/tcsonline' } },
+                                      {
+                                        'startOnlineCollectionRequest':[
+                                          { tcs_app_id : tcsAppID },
+                                          { agency_tracking_id : 'FS-TRACK-100' },
+                                          { transaction_type : 'Sale' },
+                                          { transaction_amount : response.totalCost },
+                                          { language : 'en' },
+                                          { url_success : 'http://localhost:4200' },
+                                          { url_cancel : 'http://localhost:4200' },
+                                          { account_holder_name: response.firstName + ' ' + response.lastName },
+                                          { custom_fields: [
+                                            { custom_field_1: req.body.orgStructureCode }
+                                          ]},
+                                          // TODO permitId needs to be removed before Pay.Gov integration
+                                          { permitId: response.permitId }
+                                        ]  
+                                      }
+                                    ]
+                                  }
+                                ]
+                              } 
+                            ] 
+                          } 
+                        ];
+      const xmlData = xml(xmlTemplate);
+      request.post({
+          url: vcapConstants.payGovTokenUrl,
+          method:"POST",
+          headers: {'Content-Type': 'application/xml'},
+          body: xmlData
+        },
+        function (error, response, body) {        
```

Comment:
This function should be a named function outside of the post request.

Reply:
A task has been created in #142 to pull that out into a named function.